README.md

```diff
@@ -441,17 +441,18 @@ Find districts that have a certain coverage from origin (or to destination) and
 * results: List[DistrictsResult] - The results list of districts.
 
 #### Example:
+
 ```python
 from datetime import datetime
 
 from traveltimepy import Coordinates, PublicTransport, TravelTimeSdk
 
 sdk = TravelTimeSdk('YOUR_APP_ID', 'YOUR_APP_KEY')
 
-results = sdk.districts(
```

Comment:
I'm not sure that it's worth to rename it that cause I left quite detailed readme and I was trying to avoid long names but it's not so important. Maybe it's really better to call these methods like that

Reply:
In API documentation you will find https://docs.traveltime.com/api/reference/postcode-district-filter. `postcodes_districts` is closer to API documentation naming.

Reply:
okey, let's leave like that

Reply:
postcode_districts will be better